java/fury-core/src/main/java/org/apache/fury/config/FuryBuilder.java

```diff
@@ -340,8 +340,8 @@ public Fury build() {
   }
 
   /** Build thread safe fury. */
-  public ThreadSafeFury buildThreadSafeFury() {
```

Comment:
Why remove this API? We can have a default configuration, such as use cpu cores as min instance, 2 * cpu cores as max instances

Reply:
Okay